File: Julia_Q10_Calculator.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        System.out.print("Enter the first number: ");
        double num1 = input.nextDouble();

        System.out.print("Enter the second number: ");
        double num2 = input.nextDouble();

        System.out.println("Select an operation:");
        System.out.println("1. Addition");
        System.out.println("2. Subtraction");
        System.out.println("3. Multiplication");
        System.out.println("4. Division");

        int choice = input.nextInt();

        double result;

        switch (choice) {
            case 1:
                result = num1 + num2;
                System.out.println("Result: " + result);
                break;
            case 2:
                result = num1 - num2;
                System.out.println("Result: " + result);
                break;
            case 3:
                result = num1 * num2;
                System.out.println("Result: " + result);
                break;
            case 4:
                if (num2 != 0) {
                    result = num1 / num2;
                    System.out.println("Result: " + result);
                } else {
                    System.out.println("Error: Division by zero");
                }
                break;
            default:
                System.out.println("Invalid choice");
        }

    }
}

File: Julia_Q10_Calculator.js
const num1 = parseFloat(prompt("Enter the first number:"));
const num2 = parseFloat(prompt("Enter the second number:"));

console.log("Select an operation:");
console.log("1. Addition");
console.log("2. Subtraction");
console.log("3. Multiplication");
console.log("4. Division");

const choice = parseInt(prompt("Enter your choice:"));
let result;

switch (choice) {
    case 1:
        result = num1 + num2;
        console.log(`Result: ${result}`);
        break;
    case 2:
        result = num1 - num2;
        console.log(`Result: ${result}`);
        break;
    case 3:
        result = num1 * num2;
        console.log(`Result: ${result}`);
        break;
    case 4:
        if (num2 !== 0) {
            result = num1 / num2;
            console.log(`Result: ${result}`);
        } else {
            console.log("Error: Division by zero");
        }
        break;
    default:
        console.log("Invalid choice");
}
